autorag/nodes/passagereranker/colbert.py

```diff
@@ -35,43 +35,50 @@ def colbert_reranker(queries: List[str], contents_list: List[List[str]],
     model = AutoModel.from_pretrained(model_name).to(device)
     tokenizer = AutoTokenizer.from_pretrained(model_name)
 
-    # Run async cohere_rerank_pure function
-    tasks = [get_colbert_score(query, document, model, tokenizer) for query, document, ids in
-             zip(queries, contents_list, ids_list)]
-    loop = asyncio.get_event_loop()
-    score_results = loop.run_until_complete(process_batch(tasks, batch_size=batch))
+    nested_list = [list(map(lambda x: [x], content_list)) for content_list in contents_list]
+    new_queries = list(map(lambda query, content_list: [query] * len(content_list), queries, contents_list))
```

Comment:
Why do you do this? In here, you add one more dimension at content_list and queries, and then at `colbert_run_model` you unpack those using flattened_batch_queries. 
I totally don't understand why you are doing this.
It looks colbert reranker needs 'query - list of contents' pair like any other rerankers.